README.md

```diff
@@ -1,6 +1,89 @@
 # safe-network-launch-tool
 A cross platform tool to easily launch a SAFE Network test section from home
 
+## Installing the SAFE Vault
+
+A local SAFE network is bootstrapped by running several [SAFE vaults](https://github.com/maidsafe/safe_vault) which automatically interconnect forming a network.
+
+In order to run your own local network you'd need to follow these steps:
+- Download latest release from [safe_vault releases](https://github.com/maidsafe/safe_vault/releases/tag/0.20.1)
+- Untar/unzip the downloaded file into a directory of your choice
+- Execute this tool specifying the path of the `safe_vault` executable
+
+The following is an example of how to perform this on Linux or Mac:
+```shell
+$ mkdir ~/my-local-network
+$ cd ~/my-local-network
+$ wget https://github.com/maidsafe/safe_vault/releases/download/0.21.0/safe_vault-0.21.0-x86_64-unknown-linux-musl.tar.gz
```

Comment:
Should we just use a curl command here? or if somebody tries to use these commands they will have to install the `wget` (at least on mac) before running the command which isn't mentioned here any may confuse them.

Reply:
ok, I'll change that to curl